src/app/public/modules/select-field/select-field.component.spec.ts

```diff
@@ -136,6 +146,29 @@ describe('Select field component', () => {
       expect(selectField.singleSelectPlaceholderText).toEqual('placeholder');
       expect(selectField.pickerHeading).toEqual('heading');
     });
+
+    it('should trigger blur event on touch', fakeAsync(() => {
```

Comment:
Should this test be `should trigger blur event when focus is lost`?

Reply:
done